packages/mux-player/src/index.ts

```diff
@@ -144,6 +151,7 @@ function getProps(el: MuxPlayerElement, state?: any): MuxTemplateProps {
     customDomain: el.getAttribute(MuxVideoAttributes.CUSTOM_DOMAIN) ?? undefined,
     title: el.getAttribute(PlayerAttributes.TITLE),
     novolumepref: el.hasAttribute(PlayerAttributes.NO_VOLUME_PREF),
+    extraPlaylistParams: { redundant_streams: true },
```

Comment:
NOTE: This will be made into a prop (and possibly corresponding attr) in a followup PR to allow for arbitrary query param additions, where `{ redundant_streams: true }` will be treated as the default value of the prop.

Reply:
since there is already `playbackid?token=dsdsds`
should this work similarly?

Reply:
I started there with this impl actually, but ran into issues with our state update infrastructure (bottoms out in another `...state` overwrite issue in `getProps()` that we've encountered before with other props). Happy to brainstorm together/revisit though if you'd like. Keeping it separate does feel overall cleaner/easier/less complex imo though.